ui/shared/components/panel/variants/VariantGene.jsx

```diff
@@ -370,33 +384,47 @@ const GENE_DETAIL_SECTIONS = [
     color: 'pink',
     description: 'RNA-Seq Outlier',
     label: 'RNA-Seq',
-    showDetails: (gene, rnaSeqData) => rnaSeqData && rnaSeqData[gene.geneId],
-    detailsDisplay: (gene, rnaSeqData) => (
-      <div>
-        This gene is flagged as an outlier for RNA-Seq in the following samples
-        <Table basic="very" compact="very">
-          <Table.Header>
-            <Table.Row>
-              <Table.HeaderCell />
-              {RNA_SEQ_DETAIL_FIELDS.map(
-                field => <Table.HeaderCell key={field}>{camelcaseToTitlecase(field).replace(' ', '-')}</Table.HeaderCell>,
-              )}
-            </Table.Row>
-          </Table.Header>
-          <Table.Body>
-            {Object.entries(rnaSeqData[gene.geneId]).map(([individual, data]) => (
-              <Table.Row key={individual}>
-                <Table.HeaderCell>{individual}</Table.HeaderCell>
-                {RNA_SEQ_DETAIL_FIELDS.map(
-                  field => <Table.Cell key={field}>{data[field].toPrecision(3)}</Table.Cell>,
-                )}
-              </Table.Row>
-            ))}
-          </Table.Body>
-        </Table>
-      </div>
+    showDetails: (gene, { rnaSeqData }) => rnaSeqData && rnaSeqData[gene.geneId],
+    detailsDisplay: (gene, { rnaSeqData }) => (
+      <DataTable
+        basic="very"
+        data={rnaSeqData[gene.geneId]}
+        idField="geneId"
+        columns={RNA_SEQ_COLUMNS}
+      />
     ),
   },
+  {
+    color: 'orange',
+    description: 'Phenotype Prioritization',
+    lable: 'PhenotypeGene',
+    showDetails: (gene, { phenotypeGeneScores }) => phenotypeGeneScores && phenotypeGeneScores[gene.geneId],
+    detailsDisplay: (gene, { phenotypeGeneScores }) => (Object.entries(phenotypeGeneScores[gene.geneId]).map(
+      ([tool, data]) => {
+        PHENOTYPE_GENE_SCORE_COLUMNS[tool] = [
+          ...PHENOTYPE_GENE_INFO_COLUMNS,
+          ...Object.keys(data[0].scores).map(score => ({
+            name: score,
+            content: snakecaseToTitlecase(camelcaseToTitlecase(score)).replace(' ', '-'),
```

Comment:
We use both `snakecaseToTitlecase` and `camelcaseToTitlecase` because we have both snake case and camelcase for the score names. For example, `compositeLR` and `post_test_probability`. Can we stick to just the snake case in the input file?

Reply:
I agree that it would be a good idea to process the scores when we first load them to make them all consistent. Its up to you if they should all be snake case or all be title case in the database